ddmrp_report_part_flow_index/views/stock_buffer_view.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2024 ForgeFlow S.L. (https://www.forgeflow.com)
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+<odoo>
+
+    <record id="stock_buffer_view_form_inherited" model="ir.ui.view">
+        <field name="name">stock.buffer.form - Part Flow Index</field>
+        <field name="model">stock.buffer</field>
+        <field name="inherit_id" ref="ddmrp.stock_buffer_view_form" />
+        <field name="arch" type="xml">
+            <field name="qty_multiple" position="after">
+                <field name="flow_index_group_id" />
+            </field>
```

Comment:
Can you move this to the header?

![image](https://github.com/OCA/ddmrp/assets/23449160/da6d7df9-1121-48d4-80ed-79fe30e414df)